cursorless-talon/src/actions/wrap.py

```diff
@@ -42,7 +42,7 @@ def cursorless_wrapper(m) -> dict[str, Any]:
 @mod.action_class
 class Actions:
     def cursorless_wrap(
-        action_type: str, targets: dict, cursorless_wrapper: dict[str, str]
+        action_type: str, targets: dict, cursorless_wrapper: dict[str, Any]
```

Comment:
Hmm, I have #923 open to fix up this one.

Reply:
Yes your solution is more comprehensive, but right now I just wanted to get rid of two annoying log errors. Doesn't prevent us from merging your pr.

Reply:
Sorry @auscompgeek I'm a bit behind on PRs as I was traveling. I'll try to get yours merged over the weekend